bpred.c

```diff
@@ -104,6 +104,12 @@ bpred_create(enum bpred_class class,    /* type of predictor to create */
 
     break;
 
+  case BPredPercept:
```

Comment:
I agree that the last three letters of perceptron were just too heinous to add to the name :laughing: